make/run

```diff
@@ -32,11 +32,13 @@ kubectl get storageclass persistent 2>/dev/null || {
     kubectl create -f -
 }
 
-source env/defaults.env
+DOMAIN=${DOMAIN:-$(grep '^DOMAIN=' env/defaults.env | sed 's/DOMAIN=//')}
+source <(sed '/^DOMAIN=/d' env/defaults.env)
 helm install helm \
      --namespace "${NAMESPACE}" \
      --set "env.DOMAIN=${DOMAIN}" \
-     --set "env.UAA_ADMIN_CLIENT_SECRET=${UAA_ADMIN_CLIENT_SECRET}"
+     --set "env.UAA_ADMIN_CLIENT_SECRET=${UAA_ADMIN_CLIENT_SECRET}" \
+     --set "kube.external_ip=$(getent hosts ${DOMAIN} | awk '{ print $1; exit }')"
```

Comment:
How portable is the `getent` command ? Does OSX have it ? Will have to ask @jandubois here